projects/groceries/submissions.csv

```diff
@@ -1,3 +1,3 @@
 github_username, repository_url
-my_user, https://github.com/my_user/my_repo
+NarcissuYY, https://github.com/NarcissuYY/nyu-info-2335-70-201706
```

Comment:
@NarcissuYY Sorry if the instructions weren't clear, but please create a separate GitHub repository to contain your work, and then submit the URL to that repository here next time you submit.